.github/workflows/uat.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+name: OTF UATS
+
+on:
+  pull_request:
+    branches: 'main'
```

Comment:
Running only on PRs to main for now. We can decide if we want to change this later